config.go

```diff
@@ -582,9 +582,15 @@ func loadConfig() (*config, []string, error) {
 	// Add default wallet port for the active network if there's no port specified
 	cfg.WalletHosts = normalizeAddresses(cfg.WalletHosts, activeNetParams.WalletRPCServerPort)
 
-	if len(cfg.WalletHosts) < 2 {
-		str := "%s: you must specify at least 2 wallethosts"
-		err := fmt.Errorf(str, funcName)
+	// Check if deprecated minservers config option is set
+	if cfg.MinServers != 0 {
+		str := "%s: Config minservers is deprecated.  Please remove from your config file"
+		log.Warnf(str, funcName)
+	}
+
+	if len(cfg.WalletHosts) < minRequiredBackendServers {
```

Comment:
There is already a check for > 0. 
https://github.com/decred/dcrstakepool/blob/0a62ba5d4a3712b1085eaeb4d10f4ee2393b3276/config.go#L546

The testnet and simnet checks won't matter at 1.  And @jholdstock still wants an argument as to why testnet should be different that mainnet.

Reply:
I think keeping both checks is reasonable. It allows the error output to be more detailed and more useful for the reader

Reply:
ok. After some soul searching, I am also ok with this change.